keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,108 @@
+from keras import backend
+from keras.engine import base_layer
+from keras.engine import base_preprocessing_layer
+from keras.layers.preprocessing import preprocessing_utils as utils
+from keras.utils import image_utils
+from keras_cv.utils import preprocessing
+from keras.utils import tf_utils
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.util.tf_export import keras_export
+from tensorflow.tools.docs import doc_controls
```

Comment:
@frostbyte012 

- `keras` should be imported from `tensorflow`, for example: `from tensorflow.keras import ....`
- remove not-used packages, for ex. `keras_export`.

Reply:
Agreed!  Thanks @innat

Reply:
Sure @LukeWood and @innat sir! I'm removing the unwanted packages and also adding the correct format for keras import .